geniza/footnotes/models.py

```diff
@@ -132,8 +132,12 @@ class Meta:
         ordering = ["title", "year"]
 
     def __str__(self):
-        """strip HTML tags and trailing period from formatted display"""
-        return strip_tags(self.formatted_display(extra_fields=False))[:-1]
+        # Append volume for unpublished (e.g. typed texts)
+        if self.source_type.type == "Unpublished" and self.volume:
+            return self.formatted_stripped() + ", " + self.volume
```

Comment:
I think previously volume was in parentheses, but IDK how much that matters.

Might be good to add a comment here that the string method is for internal/data admin use and the display method should be used for public display?

Reply:
Done in af1b073 👍